GotExplorer.API/Program.cs

```diff
@@ -141,12 +145,28 @@ public static void Main(string[] args)
             app.UseAuthentication();
             app.UseAuthorization();
 
-
             app.MapControllers();
 
             app.MapFallbackToFile("/index.html");
 
+            using (var scope = app.Services.CreateScope())
+            {
+                CreateRoles(scope.ServiceProvider).Wait();
+            }
             app.Run();
         }
+
+        public static async Task CreateRoles(IServiceProvider serviceProvider)
+        {
+            var roles = serviceProvider.GetRequiredService<IConfiguration>().GetSection("Roles").Get<string[]>();
+            var roleManager = serviceProvider.GetRequiredService<RoleManager<UserRole>>();
+            foreach (var role in roles)
+            {
+                if (!await roleManager.RoleExistsAsync(role))
+                {
+                    await roleManager.CreateAsync(new UserRole(role));
+                }
+            }
```

Comment:
Is not async actions exist ? 
There is not reason to use async method and wait action below